js/modules/k6/experimental/tracing/trace_id.go

```diff
@@ -0,0 +1,68 @@
+package tracing
+
+import (
+	"crypto/rand"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+)
+
+const (
+	// Being 075 the ASCII code for 'K' :)
+	k6Prefix = 0o756
+
+	// To ingest and process the related spans in k6 Cloud.
+	k6CloudCode = 12
+
+	// To not ingest and process the related spans, b/c they are part of a non-cloud run.
+	k6LocalCode = 33
+
+	// metadataTraceIDKeyName is the key name of the traceID in the output metadata.
+	metadataTraceIDKeyName = "trace_id"
+)
+
+// TraceID of 16 bytes (128 bits) are supported by w3c, b3 and jaeger
```

Comment:
This struct is 88 bits. I know we are encoding it as into a buffer of size 16 bytes, but this comment can be confusing.